maintainer.md

```diff
@@ -0,0 +1,46 @@
+# Maintaining Gotham
+
+This is documentation for maintainers of Gotham.
+This doc covers best practices and procedures on what to do for certain maintaining tasks.
+
+
+## Making Releases
+
+### Major & Minor Releases
+
+Checklist for major/minor releases:
+
+1. Make sure that the milestone is complete.
+1. Create a new branch called `release-vx.y.0`.
+1. Edit `common/hugo/version-gotham.go`. Make sure that the version number is correct and "dev" is removed.
+1. Create a full commit message: `git commit` (no `-m`)
+1. The commit title should have the format: `Release: Gotham vx.y.0`
+1. The first paragraph should be a friendly message summarizing the release.
+1. If appropriate, a list of highlights from the release should be done next.
+1. The final paragraph/line should be the output of that version's `gotham version --type=detailed`
```

Comment:
This should say:

The final paragraph/line should be the output of that version's `gotham version` or `gotham version --type=regular`

Reply:
Fixed